dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataSourceServiceImpl.java

```diff
@@ -385,6 +385,15 @@ public Result<Object> checkConnection(DbType type, ConnectionParam connectionPar
             }
             return result;
         }
+        if (type == DbType.ZEPPELIN) {
```

Comment:
Zeppelin is not a db datasource. Is there a better way to define the type?

Reply:
If you plan to improve this later when merging two modules and refactoring related code, you should add a `TDOD` comment here at least : )